### DISCUSSION TITLE
Icon Request: fa-copyleft

### DISCUSSION THREAD

**Discussion by mitchdowney** [FortAwesome/Font-Awesome#21155] [Icon Requests]
Unicode has a copyright symbol, but to my knowledge unicode does not have a copyleft symbol. It would be nifty to put copyleft symbols next to my open source projects like the copyright symbol.

**Comment by mitchdowney** [comment]
Or is there a unicode or font awesome symbol I am not aware to denote free/open source status? Thanks

**Comment by RileyStarlight** [comment]
+1

**Comment by martgnz** [comment]
It would be awesome!

**Comment by R4wizard** [comment]
+1. This would be a great addition. I just found myself needing a copyleft icon, I searched for a unicode version and found nothing. I'm wary of using &copy; and mirroring with CSS.

**Comment by marcospeebles** [comment]
All you need guys & gals  is

``` html
<i class="fa fa-copyright fa-flip-horizontal"></i>
```

:)

(edit by tagliala: fixed html code)

**Comment by valioDOTch** [comment]
+1

For balance there should be not only a copyright, but also a copyleft icon

Admittetly there's alredy a possibility to work around that as stated above
But then that was true for fa-share (fa-reply) as well and it's not an arbitrary, obscure icon that just got invented or so

**Comment by phase** [comment]
+1

**Comment by CousCousPlayer** [comment]
+1

**Comment by glueball** [comment]
+1

**Comment by illwieckz** [comment]
+1

**Comment by selmo47** [comment]
+1

**Comment by sjug** [comment]
+1

**Comment by jooola** [comment]
+1

**Comment by pablinhob** [comment]
+1

**Comment by ruthadelia** [comment]
+1

The workaround won't work for my purposes.

**Comment by Jeffrey-Polansky** [comment]
In simple terms, agreeing with marcospeebles, I just flip the copyright symbol.

**Comment by sensibleworld** [comment]
This is a quick update to let folks know that we’re refining our icon request workflow behind the scenes, and as part of that we’re going through all open requests and filing them into several new buckets to be addressed.

In this case we’ve added this issue to our new list to be created! We can’t promise any firm timelines right now, but this will help us keep track of it.

We’ll update this issue when the icon is released into the wild 🙂
